deployments/infrastructure-env-production.nix

```diff
@@ -37,5 +37,147 @@ with (import ./../lib.nix);
       hydra-ip = { inherit region accessKeyId; };
       cardanod-ip = { inherit region accessKeyId; };
     };
+    datadogMonitors =
+      let
+        appKey = fileContents ./../static/datadog-application.secret;
+        apiKey = fileContents ./../static/datadog-api.secret;
+        baseMonitorOptions = {
+          renotify_interval = 60;
+          include_tags = true;
+          no_data_timeframe = 10;
+          notify_audit = true;
+          notify_no_data = false;
+          require_full_window = true;
+        };
+        messages = {
+          critical = "notify @pagerduty-Datadog_Critical";
+          non_critical = "notify @pagerduty-Datadog_Non-Critical";
+        };
+      in
+      {
+        # ---------------------------------------------------------------------------
+        # cardano-node process
+        #
+
+        crit_cardano_node_process_monitor = { config, ... }:
+          {
+            inherit apiKey appKey;
+
+            name = "Critical: cardano-node process is down";
+            type = "metric alert";
+            message = messages.critical;
+            query = "\"process.up\".over(\"cardano-node\",\"process:cardano-node\").by(\"host\",\"process\").last(2).count_by_status()";
```

Comment:
What's the interval over this will trigger? If we restart the nodes, will it go off? It should have trigger only after 5min of downtime.